scripts/brl.sh

```
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+# Support both Yarn (INIT_CWD) and pnpm (inherited from run-with-pkg-dir or PWD)
+INIT_CWD="${INIT_CWD:-$PWD}"
+cd "$INIT_CWD" || { echo "Failed to change directory to $INIT_CWD"; exit 1; }
+
+# Function to check if index.tsx exists and run barrelsby if it doesn't
+run_barrelsby() {
+    local dir="$1"
```

Shellcheck (0.11.0)

[warning] In POSIX sh, 'local' is undefined.

(SC3043)